ffi/swig/FfiWrapper.i

```diff
@@ -50,13 +52,13 @@ import java.nio.file.StandardCopyOption;
                         StandardCopyOption.REPLACE_EXISTING);
                     System.load(file.getAbsoluteFile().toString());
                 }
-            } else {
-                throw new RuntimeException("Native code library failed to load");
+            } catch (UnsatisfiedLinkError e) {
```

Comment:
We need to revisit this.  `hasNativeLibrary()` reports that there is a file with the expected name, in the expected path.  We don't know that this file is readable or executable for our EUID, and we don't know that `System.load()` will find the file to be generally acceptable.  We also fear that something may have gone wrong with the swig-build-process and `libFfiWrapper.so` is not accessible to our thread's classLoader.  Least likely, but also possible, is that `createTempFile()` fails, due to lack of disk space, or our security context does not permit our thread to create temp files.

Another possibility we should cover is the try block throws something other than `UnsatisfiedLinkError` or `IOException`.  I know the docs say this should not happen - i just don't have confidence that this code wont be built and run under a future JRE, where other things are thrown, and let them slip through.

I don't really want our code to 'bring down the house' because we failed to catch something.

Also to re-discuss, consider the value of writing to stderr, what weird glitch we may have detected that prevents us from connecting to libvoidstar.so when there was indeed a file with that name, at the expected path at runtime